workspaces/simon-game/src/app/components/App.tsx

```diff
@@ -1,8 +1,26 @@
 import React from "react";
+import { buttonSoundConfig } from "../constants.ts";
+import playSound from "../util/playSound.ts";
 
 export function App() {
+  let audioContext: AudioContext | null = null;
   return (
     <div>
+      {buttonSoundConfig.frequencies.map((freq, index) => (
+        <button
+          key={index}
+          onClick={() =>
+            playSound(
+              (audioContext ??= new AudioContext()),
+              freq,
+              buttonSoundConfig.duration,
+              buttonSoundConfig.volume,
+            )
+          }
+        >
+          {freq}
+        </button>
+      ))}
```

Comment:
You may want to rebase, since I believe @DominicS99 landed #481.

Reply:
ack!